tests/tracing/test_sampling.py

```diff
@@ -41,26 +41,12 @@ def test_no_double_sampling(sentry_init, capture_events):
     sentry_init(traces_sample_rate=1.0, sample_rate=0.0)
     events = capture_events()
 
-    with start_transaction(name="/"):
+    with start_span(name="/"):
         pass
 
     assert len(events) == 1
 
 
-@pytest.mark.parametrize("sampling_decision", [True, False])
```

Comment:
no longer true, since unsampled spans can't have attributes and plus I don't see what the point of this test is